packages/js/email-editor/src/components/styles-sidebar/panels/typography-element-panel.tsx

```diff
@@ -111,26 +112,76 @@ export function TypographyElementPanel( {
 
 	const setLetterSpacing = ( newValue ) => {
 		updateElementStyleProp( [ 'typography', 'letterSpacing' ], newValue );
+		recordEvent(
+			'styles_sidebar_screen_typography_element_panel_set_letter_spacing',
+			{
+				element,
+				newValue,
+				selectedDefaultLetterSpacing: newValue === defaultLetterSpacing,
+			}
+		);
 	};
 
 	const setLineHeight = ( newValue ) => {
 		updateElementStyleProp( [ 'typography', 'lineHeight' ], newValue );
+		recordEvent(
+			'styles_sidebar_screen_typography_element_panel_set_line_height',
+			{
+				element,
+				newValue,
+				selectedDefaultLineHeight: newValue === defaultLineHeight,
+			}
+		);
 	};
 
 	const setFontSize = ( newValue ) => {
 		updateElementStyleProp( [ 'typography', 'fontSize' ], newValue );
+		recordEvent(
+			'styles_sidebar_screen_typography_element_panel_set_font_size',
+			{
+				element,
+				headingLevel,
+				newValue,
+				selectedDefaultFontSize: newValue === defaultFontSize,
+			}
+		);
 	};
```

Comment:
These events could be logged multiple times when the user uses range control or up/down arrows on the keyboard. I think in this place we want to track that user adjusted font size and logging multiple events when user manipulate range control would skew our data. Am I correct?
I recorded a video how events are tracked.

https://github.com/user-attachments/assets/fc0ee4fc-eea7-4930-8eed-f76b001a3819

Reply:
Fixed in https://github.com/mailpoet/mailpoet/pull/6003/commits/15bd07f23341807f82ad61f9246c18d82064fbec